app/views/crops/_predictions.html.haml

```diff
@@ -0,0 +1,29 @@
+- unless crop.perennial.nil?
+  %p
+    #{crop.name} is
+    - if crop.perennial == true
+      = link_to 'https://en.wikipedia.org/wiki/Annual_vs._perennial_plant_evolution' do
+        a perennial crop
+      (living more than two years)
```

Comment:
"two or more years"?

Reply:
btw i copied the text from the wikipedia link.

Reply:
*follows link*

OK, so a plant that lives for two years is a "biennial". We should probably represent that possibility.

Reply:
Biennials are harvested exactly like an annual but need 2 years to produce seeds. 
Carrots, beets, turnips, parsnips, broccoli, kale, cabbage, onions, collards, cauliflower, are all biennial plants. We harvest within a year, but if you leave them they'll make seed next year.

I think, for the purpose of representing these on Growstuff, "perennial=false" can mean it's annual or biennial.